__generated/rr/rrweb/rr.js

```diff
@@ -4648,7 +4648,7 @@ var CanvasManager = class {
         return hadLoadingTime;
       };
       const promises = [];
-      promises.push(...getCanvas(timestamp).filter(filterElementStartTime).map((canvas) => this.snapshot(canvas)));
+      promises.push(...getCanvas(timestamp).filter(filterElementStartTime).map(this.snapshot));
```

Comment:
this is reverting the latest rrweb fix - i think you committed rrweb unintentionally before updating it in your branch